app.rb

```diff
@@ -0,0 +1,137 @@
+require './person'
+require './student'
+require './teacher'
+require './books'
+require './rental'
+
+class App
+  def run
+    @persons = []
+    @books = []
+    puts "\nWelcome to the School Library App!"
+    display_menu
+  end
+
+  def display_menu
+    puts "\nPlease choose an option by entering a number:"
+    puts "\n1 - List all books"
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
```

Comment:
- [ ] When I type 'Create a rental' it doesn't throw the next questions to create it.

![Screen Shot 2022-03-11 at 11 24 13](https://user-images.githubusercontent.com/67211919/157916738-6d6156d1-4276-4c71-8049-e4801e4e4007.png)

Reply:
- Updated `list_all_people` now it looks like this:

  - ![listOfAllPersons](https://user-images.githubusercontent.com/83566118/158010106-0caf2d04-5208-42a9-ab98-0b4fb4a6babd.PNG)

- Updated `create_a_rental` now it looks like this:

  - ![createdRental](https://user-images.githubusercontent.com/83566118/158010190-3c135895-11d2-4bbb-baa2-86d7ff75911a.PNG)